### DISCUSSION TITLE
Jumper t18 touch screen postion inverted

### DISCUSSION THREAD

**Discussion by potatoX404** [EdgeTX/edgetx#5630] [Q&A]
I install a general 4.3-inch touch screen for my jumper t18 which had a supported touch chip gt911, all the lines are correctly welded and the FW ver has been recognized too.  But the touch position seems 180degree inverted and slightly off center, is there any way to re calibration the touch screen or I should compile a new firmware to suit the touch screen

![PXL_20241020_025038259 RAW-01 MP COVER](https://github.com/user-attachments/assets/66a01a97-b123-4062-8a6b-167743bb14e1)

**Comment by pfeerick** [comment]
The configuration registers on the GT911 could be wrong - see if this helps => https://github.com/EdgeTX/edgetx/issues/2063#issuecomment-1257155877. Short video of it in action here => https://github.com/EdgeTX/edgetx/pull/2351#issuecomment-1257157309
